worker/src/lib/dbLogger/DBLoggable.ts

```diff
@@ -757,15 +795,7 @@ function unsupportedImage(body: any): any {
   };
   if (body["image_url"] !== undefined) {
     const imageUrl = body["image_url"];
-    if (typeof imageUrl === "string" && !imageUrl.startsWith("http")) {
-      body.image_url = notSupportMessage;
-    }
-    if (
-      typeof imageUrl === "object" &&
-      imageUrl.url !== undefined &&
-      typeof imageUrl.url === "string" &&
-      !imageUrl.url.startsWith("http")
-    ) {
+    if (typeof imageUrl === "string") {
```

Comment:
Could you explain this change here?

Reply:
@colegottdank removed  !imageUrl.url.startsWith("http") to be able support base64 and keeping the same flow "Replaces all the image_url that is not a url or not { url: url }  with { unsupported_image: true }"